fe/java-udf/src/main/java/org/apache/doris/udf/JdbcExecutor.java

```diff
@@ -535,9 +557,36 @@ public void copyBatchDateTimeV2Result(Object columnObj, boolean isNullable, int
         }
     }
 
+    public String trimSpaces(String str) {
+        String a = str.trim();
```

Comment:
This implementation performance is too poor. You should implement trim by yourself